src/mqueryfront/src/auth/AuthPage.js

```diff
@@ -25,7 +25,7 @@ class AuthPage extends Component {
         const params = new URLSearchParams();
         params.append("grant_type", "authorization_code");
         params.append("code", code);
-        params.append("client_id", "app");
+        params.append("client_id", this.props.config["openid_client_id"]);
```

Comment:
Will config always have this field, event if the client doesn't use openid?

Reply:
Yes, it'll be "null" though. When login is not configured, the `"login"` button is not visible in the navbar, so user shouldn't navigate there.